### DISCUSSION TITLE
Custom domain TLS certificate expired

### DISCUSSION THREAD

**Discussion by mathiasose** [community/community#22937] [Other Feature Feedback, Questions, & Ideas]
<p>My page at <a href="https://ma.thiaso.se/" rel="nofollow noopener">https://ma.thiaso.se/</a> (alias for <a href="https://mathiasose.github.io" rel="nofollow noopener">https://mathiasose.github.io</a>) has an expired certificate. It has been expired for over 43 hours at the time of writing. Any idea why it hasn’t renewed?</p>
<p><a href="http://ma.thiaso.se/" rel="nofollow noopener">http://ma.thiaso.se/</a> (without TLS) works fine, and <a href="https://mathiasose.github.io" rel="nofollow noopener">https://mathiasose.github.io</a> redirects there.</p>
<p>As a side-note, I have never been able to select “enforce HTTPS” on the settings page, it has said " Not yet available for your site because the certificate has not finished being issued. Please allow 24 hours for this process to complete." for a long long time now.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/mathiasose">@mathiasose</a>,</p>
<p>Because this is about your specific Page and we may need to look at some things on your account to troubleshoot, I’d recommend reaching out to <a href="https://github.com/contact" rel="nofollow noopener">private support</a> for further help. They will be able to dig a little deeper and help you problem solve this.</p>
<p>Cheers!</p>
